pom.xml

```diff
@@ -565,16 +565,19 @@
         </repository>
     </repositories>
 
-    <distributionManagement>
-        <snapshotRepository>
-            <id>sonatype-nexus-snapshots</id>
-            <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
-        </snapshotRepository>
-        <repository>
-            <id>sonatype-nexus-staging</id>
-            <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
-        </repository>
-    </distributionManagement>
+
+<distributionManagement>
+    <repository>
+        <id>deploy-repo</id>
+        <name>Locally deployed releases</name>
+        <url>https://artifactory.rozkovec.name/artifactory/libs-release-local</url>
```

Comment:
Are you sure ? :-)

Reply:
Oops. That wasn't meant to be :)
Fixed.